test/WebJobs.Script.Tests.Integration/TestScripts/CustomHeadersMiddleware/CSharp/host.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": "2.0",
+  "customHeaders": {
+    "X-Content-Type-Options": "nosniff"
```

Comment:
I think this should live with the rest of the `http` settings -- @soninaren did something similar with his hsts change recently (https://github.com/Azure/azure-functions-host/pull/4506/files).

However, @soninaren -- should this live under `extensions -> http -> hsts`, rather than being at the root?

Reply:
Yeah, questionable, since this doesn't really apply to the extension.

Reply:
I did put hsts under http. I had gotten the feedback from @brettsam. It also reflects in the schema.

Reply:
Is it confusing to have two http sections? I get that it's not related directly to the extension... but I'm not sure customers would understand the difference.

Reply:
I see that @soninaren did add that to the HTTP section, so yeah, I agree it would make sense to do the same for this.